include/zephyr/drivers/firmware/nrf_ironside/ironside_dvfs.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ZEPHYR_DRIVERS_FIRMWARE_NRF_IRONSIDE_IRONSIDE_DVFS_H_
+#define ZEPHYR_INCLUDE_ZEPHYR_DRIVERS_FIRMWARE_NRF_IRONSIDE_IRONSIDE_DVFS_H_
+
+#include <stdint.h>
+#include <stddef.h>
+#include <errno.h>
+
+enum ironside_dvfs_oppoint {
+	IRONSIDE_DVFS_OPP_HIGH = 0,
+	IRONSIDE_DVFS_OPP_MEDLOW = 1,
+	IRONSIDE_DVFS_OPP_LOW = 2,
+	IRONSIDE_DVFS_OPP_COUNT
+};
```

Comment:
nit: `IRONSIDE_OPP_COUNT` is not a valid `enum ironside_dvfs_oppoint`, so it may be better to make it a macro. Maybe it could even be defined internally in the C file as `ARRAY_SIZE(dvfs_hsfll_data)`